server/lib/payments.js

```diff
@@ -301,7 +301,7 @@ export async function refundPaymentProcessorFee(
     // TODO: We're now able to support this more easily, we should implement
     const processorFeeInHostCurrency =
       processorFeeTransaction?.amountInHostCurrency || transaction.paymentProcessorFeeInHostCurrency;
-    if (refundedPaymentProcessorFee && refundedPaymentProcessorFee !== processorFeeInHostCurrency) {
```

Comment:
It's not exactly the same thing: If I get refundedPaymentProcessorFee=0 and processorFeeInHostCurrency=10, I don't want to show any warning as it is not a partial refund. The warning should only be there if `refundedPaymentProcessorFee` is truthy (> 0) **and** different than `processorFeeInHostCurrency`

Reply:
Good feedback but it's now what you think. The change is done because of the following report:

```Condition 'refundedPaymentProcessorFee' is always true at this point because it is already checked at line 290 .```